modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -298,6 +320,9 @@ export default {
         this.checked.push(doc._id);
       }
     },
+    setCheckedDocs(checkedDocs) {
+      this.checked = checkedDocs.map(doc => doc._id);
```

Comment:
This will cause checkedDocs to be reactively refilled, this dance is present because of the mixin, it makes more intuitive sense with pieces